clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1139,7 +1169,7 @@ public void pause(TopicPartition... partitions) {
     }
 
     /**
-     * Resume any partitions which have been paused with {@link #pause(TopicPartition...)}. New calls to
+     * Resume requested partitions which have been paused with {@link #pause(TopicPartition...)}. New calls to
```

Comment:
Both of these two sounds like "we are resuming all partitions that are previously paused" but we are only resuming the specified partitions that could just be a subset of the paused partitions right?